code/__defines/traits.dm

```diff
@@ -0,0 +1,64 @@
+#define TRAIT_LEVEL_EXISTS   0
+#define TRAIT_LEVEL_MINOR    1
+#define TRAIT_LEVEL_MODERATE 2
+#define TRAIT_LEVEL_MAJOR    3
+
+#define TRAITS_PHYSICAL  BITFLAG(0)
+#define TRAITS_MENTAL    BITFLAG(1)
+
+#define DEFINE_ROBOLIMB_MODEL_TRAITS(MODEL_PATH, MODEL_ID, COST) \
+/decl/trait/prosthetic_limb/left_hand/##MODEL_ID {                                                        \
```

Comment:
the spacing on the slashes here seems a bit excessive, and also they don't even seem to line up?